k8s/wordpress/chart/wordpress/templates/wordpress-manifests.yaml

```diff
@@ -23,7 +23,7 @@ metadata:
 data:
   wp-install.sh: |
     # https://codex.wordpress.org/Installing_WordPress#Installing_WordPress
```

Comment:
I think best way to wait until this port will be ready.
timeout 60 bash -c 'until echo > /dev/tcp/localhost/80; do sleep 2; done'

Reply:
I agree, looks better this way